internal/config/auth_tokens.go.old

```diff
@@ -0,0 +1,19 @@
+package config
+
+type FlashArray struct {
+        Name          string    `yaml:"name"`
+        ApiToken      string    `yaml:"api_token"`
+}
+
+type FlashArrayList struct {
```

Comment:
this seems to be a un-needed file with the file extension .old

Reply:
Deleted